docs/changelog-fragments.d/README.md

```diff
@@ -0,0 +1,76 @@
+# Adding change notes with your PRs
```

Comment:
This document shows up on GitHub when somebody navigates to the change notes directory: https://github.com/webknjaz/proxy.py/blob/maintenance/towncrier/docs/changelog-fragments.d/.
You can point the PR authors here.

Reply:
It's also included in the Sphinx docs.